src/data/items.js

```diff
@@ -0,0 +1,72 @@
+export default [
+    {
+        id: "001-beetroot",
+        name: "beetroot",
+        price: 1.80,
+        image: "assets/icons/001-beetroot.svg",
+        qty: 0
```

Comment:
Perhaps another way to deal with quantity, would be to add it as an object key, when the item is added to the cart.